### PR TITLE
Updated Layout, Added Sections, Updated Image Format, More examples

### PR DESCRIPTION
**Pull request by xhlulu**
Please review https://github.com/plotly/dash-docs/pull/98 before reviewing this one.

Additions:
* __Two Columns Layout__: Previously one column, now increases number of examples shown
* __Sections__: Examples are now classified under different section (Finance, Life sciences, Transportation...)
* __Image Format__: With shadow, smaller, text now placed after the image instead of before
* __More examples__: Added Machine Learning, Computer Vision, and Dash Components examples.

### CODE REVIEW

File: tutorial/gallery.py [hunk 1]
```diff
@@ -2,367 +2,606 @@
...
+                    src='https://github.com/plotly/dash-docs/raw/master/images/gapminder-animation.gif',
+                    alt='Screenshot of simple Dash app'
+                )
+            ),
+
+            dcc.Markdown(
+                className="markdown-links",
+                children='''
+                [View the getting started guide](https://dash.plot.ly/getting-started)
+                '''.replace('    ', '')
```
Comment (chriddyp) [inline tutorial/gallery.py]: we can use `dedent` instead of `.replace('    ', '')`

File: tutorial/gallery.py [hunk 2]
```diff
@@ -2,367 +2,606 @@
...
-            src='https://github.com/plotly/dash-docs/raw/master/images/drug-discovery-app.gif',
-            alt='Screenshot of a drug discovery Dash app'
-        )
-    ),
-
-    dcc.Markdown('''
-    [View the app](https://dash-drug-explorer.plot.ly) | [View the source code](https://github.com/plotly/dash-drug-discovery-demo/)
-
+    html.Div(className="row", children=[
+        html.Div(className="columns", children=[
```
Comment (chriddyp) [inline tutorial/gallery.py]: ⚡️ `columns` is unecessary if there isn't a number attached to it

File: tutorial/gallery.py [hunk 3]
```diff
@@ -2,367 +2,606 @@
...
+            The `animate` property in the `Graph` component was set to `True`
+            so that the points transition smoothly.
+            Some interactivity is built-in to the `Graph` component including
+            hovering over values, clicking on legend items to toggle traces, and
+            zooming into regions.
+        
+            '''.replace('    ', '')),
+        ]),
+    ]),
+    ####################################################################################################################
```
Comment (chriddyp) [inline tutorial/gallery.py]: ⚡️ - for PEP8 only comments with a single `#`

File: tutorial/gallery.py [hunk 4]
```diff
@@ -2,367 +2,606 @@
...
+                    src='https://github.com/plotly/dash-docs/raw/master/images/stock-tickers.png',
+                    alt='Screenshot of a stock tickers Dash app'
+                )
+            ),
+
+            dcc.Markdown(
+                className="markdown-links",
+                children='''
+                [Stock Tickers](https://dash-stock-tickers.plot.ly) | [Source code](https://github.com/plotly/dash-stock-tickers-demo-app)
+                '''.replace('    ', '')
```
Comment (chriddyp) [inline tutorial/gallery.py]: similarly throughout, we can use `dedent`

File: tutorial/gallery.py [hunk 5]
```diff
@@ -2,367 +2,606 @@
...
+                children='''
+            [Live Model Training Viewer](https://dash-live-model-training.plot.ly/)| [Source Code](https://github.com/plotly/dash-live-model-training)
+            '''.replace('    ', '')
+            ),
+
+            dcc.Markdown('''
+            
+            This app provides useful visualizations about what’s happening inside a complex video (e.g. object presence 
+            heatmaps, object count pie charts), rendering the data generated by the video more insightful. The data in
+            the demo is generated using MobileNet v1, trained on the COCO dataset.
```
Comment (chriddyp) [inline tutorial/gallery.py]: Let's say:
> This object-detection app provides useful visualizations about what's happening inside a complex video in real-time. The data is generated using MobileNet v1, trained on the COCO dataset. The video is displayed using an `html.Iframe` but the community-maintained [video component](https://community.plot.ly/t/modifying-a-dom-property-in-html-video-dash-video-component/7649) could also be used.

**Comment by chriddyp** [comment]
Looking good! I left a few small code-style comments

File: tutorial/gallery.py (hunk 5 above)
Comment (xhlulu) [reply tutorial/gallery.py]: The [video component](https://community.plot.ly/t/modifying-a-dom-property-in-html-video-dash-video-component/7649) is used in the app for time retrieval.

**Comment by xhlulu** [comment]
@bcdunbar I'm currently updating the gallery with a two columns format. Since it'd be really hard to merge the changes because of difference in structure changes, I directly added your commits from https://github.com/plotly/dash-docs/pull/92/ to this PR, with the updated format. Let me know if this is OK.

**Comment by bcdunbar** [comment]
@xhlulu brilliant!

**Comment by xhlulu** [comment]
@chriddyp Anything else to add?

**Comment by chriddyp** [comment]
💃 
looks good, I'll merge this in. Thanks @xhlulu !

**Pull request merged**